crates/bevy_ecs/src/system/commands/parallel_scope.rs

```diff
@@ -14,7 +14,11 @@ struct ParallelCommandQueue {
     thread_queues: Parallel<CommandQueue>,
 }
 
-/// An alternative to [`Commands`] that can be used in parallel contexts, such as those in [`Query::par_iter`](crate::system::Query::par_iter)
+/// An alternative to [`Commands`] that can be used in parallel contexts, such as those
+/// in [`Query::par_iter`](crate::system::Query::par_iter).
+/// This increases per-command overhead, and is suited to computation-heavy tasks. By
+/// contrast, [`World::spawn_batch`] collects all the work to be done and dramatically
+/// reduces the amount of computation, allocations and time required.
```

Comment:
I know this is what @alice-i-cecile said in the issue, but I don't see how `ParallelCommands` increase the overhead per-command.

This should behave exactly as `Commands` except we need to merge commands from multiple threads before applying them.
This step moves memory around, so it scales with amount of commands, but the overhead should be negligible.

The actual problem is using many `spawn` instead of one `spawn_batch`, but this is true for both `Commands` and `ParallelCommands`.

So if I can get a confirmation on this from @alice-i-cecile,
I suggest keeping the original comment here and putting the relevant documentation in `spawn` commands (pointing to `spawn_batch`).

Reply:
Yeah, let's cut the discussion on per-command overhead and focus on a) please benchmark your perf optimizations and b) spawn_batch is the right tool for spawning many entities.

We *could* bench it to get a real answer, but that's liable to change over the releases.